src/prefect/server/services/perpetual_services.py

```diff
@@ -0,0 +1,242 @@
+"""
+Perpetual services are background services that run on a periodic schedule using docket.
+
+This module provides the registry and scheduling logic for perpetual services,
+replacing the LoopService pattern with docket's Perpetual dependency.
+
+Key outcomes preserved from LoopService:
+- Services can be enabled/disabled via settings (PREFECT_SERVER_SERVICES_<NAME>_ENABLED)
+- Services respect RunInEphemeralServers/RunInWebservers markers
+- Services are only scheduled when enabled (disabled services never run)
+- Automatic Perpetual functions are scheduled at worker startup
+- Non-automatic Perpetual functions are explicitly scheduled based on settings
+"""
+
+from __future__ import annotations
+
+import logging
+from dataclasses import dataclass
+from typing import TYPE_CHECKING, Any, Awaitable, Callable, TypeVar
+
+from docket import Docket, Perpetual
+from docket.dependencies import get_single_dependency_parameter_of_type
+
+from prefect.logging import get_logger
+
+if TYPE_CHECKING:
+    from prefect.settings.models.server.services import ServicesBaseSetting
+
+logger: logging.Logger = get_logger(__name__)
+
+# Type aliases for perpetual service functions and getters
+PerpetualServiceFunction = Callable[..., Awaitable[Any]]
```

Comment:
If you want to unify with Docket's definition: `from docket.execution import TaskFunction`

https://github.com/chrisguidry/docket/blob/main/src/docket/execution.py#L36